docs/setup-cluster/security/tls.rst

```diff
@@ -75,3 +75,247 @@ untrusted certificate and display a hash of the certificate. You may wish to con
+
+Certificate Creation When the Determined Master is Behind a VPN
+---------------------------------------------------------------
+
+This section provides information about requesting the Let's Encrypt certificate in environments
+that do not provide inbound access from Let’s Encrypt to port Determined Master server port 80
+(e.g., when Determined Master is behind a VPN).
+
+Request the Let's Encrypt Certificate using the DNS-01 Challenge
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following command to request the Let's Encrypt certificate using the DNS-01 challenge domain
+verification:
+
+.. code:: bash
+
+   # certbot certonly --manual --preferred-challenges dns -d <domain>
+
+Certbot responds and lets you know that before continuing you should verify the TXT record has been
+deployed:
+
+.. code::
+
+   Saving debug log to /var/log/letsencrypt/letsencrypt.log
+   Requesting a certificate for <domain>
+
+   Please deploy a DNS TXT record under the name:
+
+   _acme-challenge.<domain>.
+
+   with the following value:
+
+   <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX domain token>
+
+   Before continuing, verify the TXT record has been deployed. Depending on the DNS
+   provider, this may take some time, from a few seconds to multiple minutes. You can
+   check if it has finished deploying with the aid of online tools, such as the Google
+   Admin Toolbox: https://toolbox.googleapps.com/apps/dig/#TXT/_acme-challenge.<domain>.
+   Look for one or more bolded line(s) below the line ';ANSWER'. It should show the
+   value(s) you've just added.
+
+   Press Enter to Continue
+
+.. caution::
+
+   DO NOT PRESS ENTER BEFORE SETTING UP THE DNS RECORD
+
+Set Up the DNS Record Before Pressing ``ENTER``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+DNS TXT Record example
+
++---------------------------------+-------------+-----+-------------------------------------------------+
+| FQDN                            | RECORD TYPE | TTL | Value                                           |
++=================================+=============+=====+=================================================+
+| _acme-challenge.<domain>.       | TXT         | 900 | <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  |
+|                                 |             |     | domain token>                                   |
++---------------------------------+-------------+-----+-------------------------------------------------+
+
+Ensure the ``_acme-challenge.<domain>.`` DNS record has been propagated using one of the following:
+
+-  ``https://toolbox.googleapps.com/apps/dig/#TXT/_acme-challenge.<domain>.``, or
+-  ``nslookup -type=TXT _acme-challenge.<domain>.``
+
+.. note::
+
+   You may need to install bind-utils to run ``nslookup`` using the following command:
+
+   .. code:: bash
+
+      yum install bind-utils
+
+Press ``ENTER``
+^^^^^^^^^^^^^^^
+
+Once you have set up the DNS record, press **Enter**.
```

Comment:
```suggestion
Once you have set up the DNS record, press Enter.
```

Reply:
Where a user interacts with the name of the command, the commands is in boldface type.